### PR TITLE
refactor(header) : replace header scss module with material make styles service component 

### PR DESCRIPTION
**Pull request by Noha-ElAbrouki**
replace header scss module with material make styles for service component

### CODE REVIEW

File: www/front_src/src/Header/RessourceStatusCounter/Service/index.tsx [hunk 1]
```diff
@@ -107,7 +106,7 @@ const ServiceStatusCounter = (): JSX.Element => {
       schema={statusSchema}
     >
       {({ hasPending, data, toggled, toggleDetailedView }): JSX.Element => (
-        <div className={`${styles.wrapper} wrap-right-services`}>
```
Comment (jdelpierre) [inline www/front_src/src/Header/RessourceStatusCounter/Service/index.tsx:110]: `wrap-right-services`it is not really used anywhere?

Comment (Noha-ElAbrouki) [reply www/front_src/src/Header/RessourceStatusCounter/Service/index.tsx:110]: yes , here is an image for this component
serviceComponent

Comment (jdelpierre) [reply www/front_src/src/Header/RessourceStatusCounter/Service/index.tsx:110]: Yes but I see it has been completely removed, so I don't know if it was already useless before the PR (not used anywhere else but here)

Comment (Noha-ElAbrouki) [reply www/front_src/src/Header/RessourceStatusCounter/Service/index.tsx:110]: it's the dark mode branch, I didn't  see the effect so i deleted it ?
wrapRightUser

Comment (jdelpierre) [reply www/front_src/src/Header/RessourceStatusCounter/Service/index.tsx:110]: I agree, it is quite possible that this is obsolete.

**Review by jdelpierre — APPROVED** [review]

File: www/front_src/src/Header/RessourceStatusCounter/index.tsx [hunk 2]
```diff
@@ -11,10 +11,28 @@ import { refreshIntervalAtom } from '@centreon/ui-context';
...
+    backgroundColor: theme.palette.common.black,
+    boxSizing: 'border-box',
+    display: 'none',
+    left: theme.spacing(0),
+    padding: theme.spacing(1),
+    position: 'absolute',
+    textAlign: 'left',
+    top: '100%',
+    width: '100%',
+    zIndex: 99,
```
Comment (Thebarda) [inline www/front_src/src/Header/RessourceStatusCounter/index.tsx:28]: Is it possible to use the `theme.zIndex.mobileStepper` https://mui.com/customization/default-theme/

**Review by bdauria — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**